packages/jsii-pacmak/test/expected.jsii-calc/python/src/jsii_calc/submodule/back_references/__init__.py

```diff
@@ -1,23 +1,18 @@
 import abc
 import builtins
-import datetime
-import enum
-import typing
 
 import jsii
 import jsii.compat
 import publication
 
-import scope.jsii_calc_base
-import scope.jsii_calc_base_of_base
-import scope.jsii_calc_lib
-
 from ..._jsii import *
 
+from .. import (MyClass as _MyClass_a2fdc0b6)
+
 
 @jsii.data_type(jsii_type="jsii-calc.submodule.back_references.MyClassReference", jsii_struct_bases=[], name_mapping={'reference': 'reference'})
 class MyClassReference():
-    def __init__(self, *, reference: jsii_calc.submodule.MyClass):
+    def __init__(self, *, reference: _MyClass_a2fdc0b6) -> None:
```

Comment:
Is it possible, instead of these hashified aliases, to always import from the root? So this would be:

```py
from ... import submodule
```

And then the usage would be:

```py
submodule.MyClass
```

Reply:
That does not work when you import from the root itself... In this case, I need to always alias... And it makes the code simpler to always do the same, rather than ad yet another branch...

Reply:
Oh and it also breaks down if we import something named `submodule` from two different packages (for example, one local, one from a dependency).

Reply:
Why do you need to alias? Can't you do `from . import Type` or something?

Reply:
`submodule` can't be a type name (all lowercase), only a module name, no?

Reply:
That won't save me when I have the same submodule "leaf name" at different points...

```
from .foo import submodule
from .bar import submodule

submodule.FancyType # <-- Which "submodule" is this?
```

And while a submodule name cannot be a type name, I can have the same type name defined in several submodules:

```
from .foo import MyType
from .bar import MyType

MyType() # <-- Which is that?
```

When using relative imports, once has no choice but to use the `from .blah import thing`, and that appears to prevent the use fo the fully qualified type name further down (which would have been possible with an absolute import of the form `import foo.bar`). Unfortunately, I have to bring types into the present namespace, which means I have to deal with collisions myself, too.